test/e2e/authorization.go

```diff
@@ -642,6 +643,19 @@ var _ = g.Describe("Authorization via admission-controller [RBAC] [Zalando]", fu
 				framework.ExpectNoError(err, "failed to create pod: %s in namespace: %s", nonSystemResource.Name, nonSystemResource.Namespace)
 			})
 
+			// Not needed actually
```

Comment:
I commented this out because the missing test cases were about the namespaces itself rather than resources within the namespace.

But we could also add this test case. We would just need to create a resource first, otherwise the delete call doesn't work correctly.